src/algebraic_geometry/stalks.lean

```diff
@@ -100,6 +100,58 @@ begin
   erw [category_theory.functor.map_id],
   erw [id_comp, id_comp, id_comp],
 end
+
+/--
+If `α = β` and `x = x'`, we would like to say that `stalk_map α x = stalk_map β x'`.
+Unfortunately, this equality is not well-formed, as their types are not _definitionally_ the same.
+To get a proper congruence lemma, we therefore have to introduce these `eq_to_hom` arrows on
+either side of the equality.
```

Comment:
This is quite unfortunate. But I don't see a better solution atm.

Reply:
Yes, it took me some time to get used to all the eq_to_homs involved... But now I think it's not so bad after all.